_includes/docs/latest/ajax-api.html

```diff
@@ -1130,6 +1128,90 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
 }
 </pre>
 
+<h3 id="api-schedule-a-flow">Flexible scheduling using cron</h3>
+
+<p>
+This API call schedules a flow by a cron Expression. Cron is a UNIX tool that has been widely used for a long time, and we use <a href="http://www.quartz-scheduler.org/">Quartz library</a> to parse cron Expression. All cron schedules follow the timezone defined in azkaban web server (the timezone ID is obtained by <i>java.util.TimeZone.getDefault().getID()</i>).
+</p>
+<ul class="list-unstyled">
+  <li>
+    <strong>Method:</strong> POST
+  </li>
+  <li>
+    <strong>Request URL:</strong> /schedule?ajax=scheduleCronFlow
+  </li>
+  <li>
+    <strong>Parameter Location:</strong> Request Query String
+  </li>
+</ul>
+
+<h5><strong>Request Parameters</strong></h5>
+
+<table class="table table-striped table-condensed table-bordered">
+  <thead>
+    <tr>
+      <th>Parameter</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>session.id</td>
+      <td>The user session id.</td>
+    </tr>
+    <tr>
+      <td>ajax=scheduleCronFlow</td>
+      <td>The fixed parameter indicating the action is to use cron to schedule a flow.</td>
+    </tr>
+    <tr>
+      <td>projectName</td>
+      <td>The name of the project.</td>
+    </tr>
+    <tr>
+      <td>flowName</td>
+      <td>The name of the flow.</td>
+    </tr>
+    <tr>
+      <td>cronExpression</td>
+      <td>A CRON expression is a string comprising 6 or 7 fields separated by white space[8] that represents a set of times. In azkaban, we use <a href="http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html"> Quartz Cron Format</a>.</td>
+    </tr>
+  </tbody>
+</table>
+
+<p>Here's a curl command sample:</p>
+
+<pre class="code">
+curl -k https://HOST:PORT/schedule -d "ajax=scheduleCronFlow&projectName=PROJECT_NAME&flow=FLOW_NAME&cronExpression=0 * 5,7-10 ? * 4-6" -b azkaban.browser.session.id=SESSION_ID
```

Comment:
Does the expression need to be URL encoded? Do you need to pass the additional parameter? 
see

"To URL-encode the value of a form field you may use --data-urlencode."
https://curl.haxx.se/docs/manpage.html